qweb_usertime/README.rst

```diff
@@ -14,24 +16,37 @@ language::
 
     <t t-usertime="" />
 
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to ...
 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/9.0
 
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
-In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/server-tools/issues/new?body=module:%20qweb_usertime%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
-
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```

Comment:
@GillesTephaneMeyomesse Complete {project_repo}, they are in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt